mne/source_estimate.py

```diff
@@ -1370,18 +1370,14 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         hemi : str, 'lh' | 'rh' | 'split' | 'both'
             The hemisphere to display. Using 'both' or 'split' requires
             PySurfer version 0.4 or above.
-        colormap : str
-            The type of colormap to use.
+        colormap : str | array
```

Comment:
please go over your docs and do something like this! We've never been that close!